scripts/cli/build_cli_release.ps1

```diff
@@ -11,10 +11,16 @@
 $NAME="aptos-cli"
 $CRATE_NAME="aptos"
 $CARGO_PATH="crates\$CRATE_NAME\Cargo.toml"
+$VCPKG_ROOT="C:\vcpkg"
 
 # Get the version of the CLI from its Cargo.toml.
 $VERSION = Get-Content $CARGO_PATH | Select-String -Pattern '^\w*version = "(\d*\.\d*.\d*)"' | % {"$($_.matches.groups[1])"}
 
+# Note: This is required to bypass openssl isssue on Windows.
+echo "Setting up vcpkg"
+echo "VCPKG_ROOT=$VCPKG_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
+vcpkg install openssl:x64-windows-static-md --clean-after-build
+
```

Comment:
Interesting, this actually should solve the problem how I did it let me see if I can involve it in a new build